app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -11,10 +11,12 @@
     <icon drawable="@drawable/camera" package="com.android.camera" name="Camera" />
     <icon drawable="@drawable/camera" package="com.huawei.camera" name="Camera" />
     <icon drawable="@drawable/camera" package="com.honor.camera" name="Camera" />
+    <icon drawable="@drawable/camera" package="org.lineageos.snap" name="Camera" />
     <icon drawable="@drawable/chrome_beta" package="com.chrome.beta" name="Chrome Beta" />
     <icon drawable="@drawable/chrome_dev" package="com.chrome.dev" name="Chrome Dev" />
     <icon drawable="@drawable/files" package="com.google.android.documentsui" name="Files" />
     <icon drawable="@drawable/files" package="com.android.documentsui" name="Files" />
+    <icon drawable="@drawable/files" package="com.google.android.apps.nbu.files" name="Files" />
```

Comment:
A dedicated icon for Files by Google is already included in Lawnchair itself (see image). I would suggest sticking to that rather than switching to the generic icon for system file explorers.

![image](https://user-images.githubusercontent.com/56888459/141452707-df61b513-4f0a-4da5-9815-efce1b7c0aad.png)

```suggestion
```

Reply:
Done :)